splines/splineFDBase.cpp

```diff
@@ -168,21 +171,19 @@ void splineFDBase::TransferToMonolith()
                   int iCoeff=coeffindexvec[splineindex];
                   getSplineCoeff_SepMany(splineindex, tmpXCoeffArr, tmpManyCoeffArr);
 
-                  #ifdef MULTITHREAD
-                  #pragma omp parallel for
-                  #endif
-                  for(int i=0; i<splineKnots; i++){
+                  for(int i = 0; i < splineKnots; i++){
 
                     if(tmpXCoeffArr[i]==-999){
```

Comment:
hard coded magic numbers

Reply:
these checks on -999 don't actually make any sense so I'll remove them. The spline coefficients are initialised to -999 here: https://github.com/mach3-software/MaCh3/blob/develop/splines/splineFDBase.cpp#L767
But then we immediately call GetCoef() where if the there is some failure here we would not get -999 out. So I have removed this check.